examples/breakout-rooms/src/components/BreakoutStarter/BreakoutStarter.css

```diff
@@ -11,21 +11,46 @@
 }
 
 .starter-action {
-  background: #f2f4fc;
-  border-radius: 0.25em;
-  padding: 1em;
+  background: var(--blue200);
+  border: none;
+  border-radius: var(--border-radius-medium);
+  cursor: default;
   display: flex;
+  padding: 1em;
```

Comment:
can we use mirotone spacing?

Reply:
Yes, still updating values across the branch!